versions.yaml

```diff
@@ -123,6 +123,12 @@ externals:
     url: "https://github.com/opencontainers/runc"
     version: "v1.0.0-rc5"
 
+  cri-containerd:
+    description: |
+      Containerd Plugin for Kubernetes Container Runtime Interface
+    url: "https://github.com/containerd/cri"
+    version: "v1.0.0-rc.2"
+
```

Comment:
Nit: Could you move this up to just below the `cri:` section so the externals are kept sorted?

Reply:
Updated